tests/utils/fc.rs

```diff
@@ -19,9 +19,20 @@ use super::config;
 
 // TODO: rename this function, this launches fcd, not 'clients'
 pub async fn setup_clients() -> (process::Child, Vec<String>, process::Child, Vec<String>) {
+    let (farcasterd_maker, data_dir_maker) = launch_farcasterd_maker();
+    let (farcasterd_taker, data_dir_taker) = launch_farcasterd_taker();
+    (
+        farcasterd_maker,
+        data_dir_maker,
+        farcasterd_taker,
+        data_dir_taker,
+    )
+}
+
+// TODO: rename this function, this launches fcd, not 'clients'
```

Comment:
This line can be removed, here and below, and we can do the rename of `setup_clients` in this PR IMO

Reply:
[84d91f6](https://github.com/farcaster-project/farcaster-node/pull/752/commits/84d91f6ee50460fabbe405836f8a38baa991d617)